MapboxDirections.xcodeproj/project.pbxproj

```diff
@@ -1056,8 +1056,8 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = DA1A10C41D00F8FF009F82FA /* Build configuration list for PBXNativeTarget "MapboxDirectionsMac" */;
 			buildPhases = (
-				DA1A10AA1D00F8FF009F82FA /* Sources */,
```

Comment:
While using Xcode 13.4.1 build issue sometimes occurs because of configuration list sequence.